### ISSUE TITLE
relation of defaultBorderRadius to .selection.list > .item

### ISSUE THREAD

**Issue by jholl** [Semantic-Org/Semantic-UI#3103]
in list.less

```
  .ui.selection.list .list > .item,
  .ui.selection.list > .item {
      ...
      border-radius: @selectionListItemBorderRadius;
  }
```

in default theme this is hardcoded `@selectionListItemBorderRadius: 0.5em` -- would this be something to relate to `@defaultBorderRadius` so that when applying `@defaultBorderRadius: 0em` in custom settings that selection list items also have no border radius?

**Comment by jlukic** [comment]
Yeah, thats a dark corner of code. Should be `@defaultBorderRadius`
